en/admin/db_manage.rst

```diff
@@ -57,9 +57,13 @@ The volumes of CUBRID database are classified as permanent volume, temporary vol
 *   In the permanent volumes,
 
     *   there are data volumes, that usually store permanent data, but can also store temporary data.
-    *   there are log volumes, that can be further classified as: one active log, archive logs and one background archiving log.
+    *   there is the active log volume, that includes late changes of log records.
```

Comment:
I think it seems weird. 
```suggestion
    *   there is the active log volume, that includes late changes of database.
```

Reply:
I should read or ask about the concepts about log and then I will answer this.

Reply:
A 'log record' contains changes in the database. And the set of 'log records' is the log volume. 'log record' is not changed after being logged.

Reply:
Thank you! Now I can understand why `late changes of log records` is weird sentence. So, I checked whether similar sentence exists. I will apply it thank you!